### PR TITLE
Bug fixes and confirmation messages

### LINKED ISSUE #135: Add confirmation message on adding event
image

### PR DESCRIPTION
**Pull request by JeremiasLiew**
Fix bugs with deleting events
Rewrite listUpcoming method
Add confirmation messages to add and delete events

Closes #135

### CODE REVIEW

File: src/main/java/seedu/subjects/SubjectList.java [hunk 1]
```diff
@@ -139,39 +141,76 @@ public static void listSubjects(ArrayList<Subject> subjects) {
...
 
     /**
      * Lists all upcoming events.
-     * @param events List of upcoming events.
+     * @param dateRange Date range of upcoming events to show.
      */
-    public static void listUpcoming(ArrayList<Event> events) {
-        if (events.size() == 0) {
-            System.out.println("No upcoming events.");
+    public void listUpcoming(int... dateRange) {
```
Comment (kcubey) [inline src/main/java/seedu/subjects/SubjectList.java]: Would it be better to put this & findUpcoming() into the Events package/class instead? But if this is preferable then it's ok

Comment (JeremiasLiew) [reply src/main/java/seedu/subjects/SubjectList.java]: Actually thats true. I think I should create an EventList class like the CardList class. So I can shift some methods over also

**Comment by JeremiasLiew** [comment]
In the end I stuck with your original storage, just changed everything outside it, because github didnt like how i did it.
Reposense might wrongly flag some parts of storage as mine (idk how it works), so maybe u might need to copy and paste it again to get it back psps ><

**Review by kcubey — APPROVED** [review]
Looks ok! Will verify Storage

**Pull request merged**